app/containers/App/Footer/Footer.jsx

```diff
@@ -1,7 +1,11 @@
 // @flow
 import React from 'react'
 import styles from './Footer.scss'
+import { shell } from 'electron'
 
-const Footer = () => <div className={styles.container}>Created by CoZ. Donations: Adr3XjZ5QDzVJrWvzmsTTchpLRRGSzgS5A</div>
+const discordInviteLink = 'https://discordapp.com/invite/R8v48YA'
+const openDiscordLink = () => shell.openExternal(discordInviteLink)
+
+const Footer = () => <div className={styles.container}>Discord Link: <a href='#' onClick={openDiscordLink}>{discordInviteLink}</a> | Created by CoZ. Donations: Adr3XjZ5QDzVJrWvzmsTTchpLRRGSzgS5A</div>
```

Comment:
This is great, thanks for the contribution!  Could we just change the "Discord Link" text to something like "Support" or "Community Support"?